src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -23,8 +24,10 @@
 import seedu.address.model.person.Address;
 import seedu.address.model.person.Class;
 import seedu.address.model.person.Email;
-import seedu.address.model.person.Money;
+import seedu.address.model.person.MoneyOwed;
+import seedu.address.model.person.MoneyPaid;
```

Comment:
This is erasing the changes made in pr #105. I notice that in the commit [f0dff56](https://github.com/AY2223S1-CS2103T-T09-4/tp/pull/103/commits/f0dff565ed905d169402744e40faf4a6f6ffdcf9) you have included the abstraction of Money field from MoneyOwed and MoneyPaid but then reverted back. May I know the rationale for this decision?

Reply:
Hmm yes I think it erases some previous PR. I think because she merged commit `f0dff56` (which was correct) but reverted the merge in commit `d65b8c6` which caused some lines to be deleted.